lib/Minify/Minify/Cache/ZendPlatform.php

```diff
@@ -81,17 +81,14 @@ public function isValid($id, $srcMtime)
     }
 
 
-    /**
-     * Send the cached content to output
-     *
-     * @param string $id cache id
-     */
-    public function display($id)
-    {
-        echo $this->_fetch($id)
-            ? $this->_data
-            : '';
-    }
+	/**
+	 * Send the cached content to output
+	 *
+	 * @param string $id cache id
+	 */
+	public function display( $id ) {
+		echo $this->_fetch( $id ) ? esc_html( $this->_data ) : '';
```

Comment:
Check if content needs to get escaped.

Reply:
Switched to wp_kses with call to Util_Ui::get_allowed_html_for_wp_kses_from_content for allowed HTML tags